src/Sql/Sql/help/New-AzSqlInstanceServerTrustCertificate.md

```diff
@@ -0,0 +1,216 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/powershell/module/az.sql/new-azsqlinstanceservertrustcertificate
+schema: 2.0.0
+---
+
+# New-AzSqlInstanceServerTrustCertificate
+
+## SYNOPSIS
+Creates a new server trust certificate.
+
+## SYNTAX
+
+### CreateByNameParameterSet (Default)
+```
+New-AzSqlInstanceServerTrustCertificate [-ResourceGroupName] <String> [-InstanceName] <String> [-Name] <String>
+ [-PublicKey] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
I'm wondering should public key be wrapped with securestring. https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#secret-parameters

Reply:
I would say no since this is a **public** key, it's designed to be shared

Reply:
Make sense.